### PR TITLE
Add "Raw" Links to Readme

### PR DESCRIPTION
**Pull request by kgobin2000**
#100  Added instructions on how to import collection into postman

### CODE REVIEW

**Comment by brownoxford** [comment]
@kgobin2000, thanks so much for the PR! I have one suggestion on the formatting of these instructions - In general, these tutorial README files are intended to be a walk-through for someone unfamiliar with Postman and/or the procedures involved. I would suggest taking the language that you are adding and putting it at the top of the tutorial (just under the headline), something like
> # Credentials Issue Tutorial
> If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/credentials-issue/credentials-issue.postman_collection.json) to your clipboard and In Postman, Import -> Link -> Paste the "Raw" link. Otherwise, please continue with the rest of the tutorial.
>
> ---
> If you have not done so already...

This way, it doesn't interrupt the flow of the tutorial and confuse new users by adding a bunch of functionality to which they have not yet been introduced.

**Review by nissimsan — APPROVED** [review]

**Comment by nissimsan** [comment]
+1, great to see you contribute a PRs, @kgobin2000! I've approved - will leave it to you to consider incorporating @brownoxford suggestions.

Now, since you're on a roll already :) might I suggest that you also pick up the actual aim of ticket https://github.com/w3c-ccg/traceability-interop/issues/100. It's just adding a list of those same collections to the respec.

**Comment by kgobin2000** [comment]
> @kgobin2000, thanks so much for the PR! I have one suggestion on the formatting of these instructions - In general, these tutorial README files are intended to be a walk-through for someone unfamiliar with Postman and/or the procedures involved. I would suggest taking the language that you are adding and putting it at the top of the tutorial (just under the headline), something like
> 
> > # Credentials Issue Tutorial
> > If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/credentials-issue/credentials-issue.postman_collection.json) to your clipboard and In Postman, Import -> Link -> Paste the "Raw" link. Otherwise, please continue with the rest of the tutorial.
> > 
> > If you have not done so already...
> 
> This way, it doesn't interrupt the flow of the tutorial and confuse new users by adding a bunch of functionality to which they have not yet been introduced.

Great suggestion! I will incorporate it into it into the changes.

**Comment by kgobin2000** [comment]
> +1, great to see you contribute a PRs, @kgobin2000! I've approved - will leave it to you to consider incorporating @brownoxford suggestions.
> 
> Now, since you're on a roll already :) might I suggest that you also pick up the actual aim of ticket #100. It's just adding a list of those same collections to the respec.

Definitely, I'll work on it.

**Comment by kgobin2000** [comment]
> +1, great to see you contribute a PRs, @kgobin2000! I've approved - will leave it to you to consider incorporating @brownoxford suggestions.
> 
> Now, since you're on a roll already :) might I suggest that you also pick up the actual aim of ticket #100. It's just adding a list of those same collections to the respec.

@nissimsan By adding it to the respec, do you mean adding the collections to this document? https://w3id.org/traceability/interoperability/openapi

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

File: docs/tutorials/authentication/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Authentication Tutorial
 
+If you wish to bypass this tutorial and simply want to import the postman collection locally, copy the "Raw" [link](https://raw.githubusercontent.com/w3c-ccg/traceability-interop/main/docs/tutorials/authentication/authentication.postman_collection.json) to your clipboard and In Postman, Import -> Link -> Paste the "Raw" link. Otherwise, please continue with the rest of the tutorial.
+
+#
```
Comment (nissimsan) [inline docs/tutorials/authentication/README.md:5]: Is this intetional?

**Comment by TallTed** [comment]
@kgobin2000 -- Now that I understand what the empty `#` (for `<h1>`) was used for... it should have been lines of `-----`(eol) preceded and followed by empty lines.

The empty `#` will break various indexing tools, including TOC builders, because there should be only one `<h1>` or `#` per document (and an empty TOC entry is a bear).

**Pull request merged**